src/App.js

```diff
@@ -3,11 +3,12 @@ import "./App.css";
 import { MockBee } from "./backend/mockdocs/MockBee";
 import { MockAPI } from "./backend/mockdocs/MockMan";
 import { Footer, Header, Loader } from "./components";
-import { useLoader } from "./contexts";
-import { Landing, NotFound } from "./pages";
+import { useAuth, useLoader } from "./contexts";
+import { Home, Landing, Login, NotFound, SignUp } from "./pages";
 
 function App() {
 	const { pathname } = useLocation();
+	const { isAuth } = useAuth();
 	const { loader } = useLoader();
 	const isLandingPage = (() => pathname === "/")();
```

Comment:
You could have done there something like

`const landingPageClass = pathname===" /" ? "landing": "";`

I you have done some extra work you could have avoided